src/main.rs

```diff
@@ -519,9 +569,14 @@ impl Fixer {
 
                 // This chaining is necessary because `MachOIterator::Item` is
                 // not `MachO` but `Result<MachO>`. We don't distinguish
-                // between the "couldn't find the x86-64 code" case and the
+                // between the "couldn't find the $target_arch code" case and the
                 // "found it, but it had an error" case.
-                let msg = "find x86-64 code in the fat binary";
+                #[cfg(target_arch = "x86_64")]
+                let msg = "find x86_64 code in the fat binary";
+                #[cfg(target_arch = "aarch64")]
+                let msg = "find arm64 code in the fat binary";
+                #[cfg(all(not(target_arch = "aarch64"), not(target_arch = "x86_64")))]
+                let msg = "find code in the fat binary";
```

Comment:
I don't know if this is strictly useful for this code, but I figure I'll point out in case it's useful. There's a builtin `cfg!` macro that resolves to a `bool` so you can use normal control flow:

```
    if cfg!(target_arch = "x86_64") {
        println!("hello! x64");
    } else {
        println!("hello! other");
    }
```

Usually that's nice for runtime code. e.g. you don't need to faff about with stuff like

```
 #[cfg(all(not(target_arch = "aarch64"), not(target_arch = "x86_64")))]
```